homeassistant/components/remember_the_milk.py

```diff
@@ -0,0 +1,267 @@
+"""Component to interact with Remember The Milk.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/remember_the_milk/
+
+Minimum viable product, it currently only support creating new tasks in your
+Remember The Milk (https://www.rememberthemilk.com/) account.
+
+Ideas for improvement:
+- add interface to resolve tasks
+- add option to add a task to a certain list
+- add option to list tasks stored in RTM
+- make registration dialog nicer
+"""
+import logging
+import os
+import json
+import voluptuous as vol
+from homeassistant.const import (CONF_API_KEY, STATE_OK)
+import homeassistant.helpers.config_validation as cv
+
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['RtmAPI==0.7.0', 'httplib2==0.10.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'remember_the_milk'
+DEFAULT_NAME = DOMAIN
+GROUP_NAME_RTM = 'remember the milk accounts'
+
+CONF_SHARED_SECRET = 'shared_secret'
+
+RTM_SCHEMA = vol.Schema({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_SHARED_SECRET): cv.string,
+})
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: {
+        cv.string: RTM_SCHEMA
+    }
+}, extra=vol.ALLOW_EXTRA)
+
+CONFIG_TOKEN = 'token'
+CONFIG_FILE_NAME = 'remember_the_milk.conf'
+
+SERVICE_DESCRIPTION_CREATE_TASK = {
```

Comment:
Should you put that in the `services.yaml` file? There was something about this not so long again but I can remember.

Reply:
OK